src/assets/locales/en.json

```diff
@@ -73,8 +73,19 @@
     "ok": "OK",
     "title": "Choose the time"
   },
+  "candidateList": {
+    "select": "select",
+    "search": "search",
+    "haveChosen": "You have chosen",
+    "other": "other",
+    "others": "others",
+    "perPage": "Items per page",
+    "nextPage": "Next page",
+    "prevPage": "Previous page",
+    "of": "of"
+  },
   "tostr": {
-	"title": "Sucess",
+    "title": "Sucess",
```

Comment:
Sorry, I know this is not part of this ticket, but could you also please fix spelling of the word `success`?

Reply:
no problem :)